cmake/aws-c-io-config.cmake

```diff
@@ -7,6 +7,7 @@ if (UNIX AND NOT APPLE)
 endif()
 
 find_dependency(aws-c-common)
+find_dependency(aws-c-cal)
```

Comment:
should you remove the `find_dependency(LibCrypto)` a few lines above this?

also, not sure if it matters,  but we don't *need* to have this line anymore:
`list(APPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/modules")`
That was added so downstream projects would find the `FindLibCrypto.cmake` file that we were installing there, but that file no longer exists, so probably the `/modules` folder doesn't exist either on a clean install. Not sure if this matters...